src/main/java/ladder/domain/Line.java

```diff
@@ -1,45 +1,100 @@
 package ladder.domain;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.*;
+import static ladder.domain.Direction.*;
 
 public class Line {
 
+    private static final Random random = new Random();
     private final List<Point> points;
 
     private Line(List<Point> points) {
-        Point current = points.get(0);
+        for (int i = 0 ; i < points.size() - 1 ; i++) {
```

Comment:
```suggestion
        for (int i = 0 ; i < points.size() - 1 ; i++) {
```
points.size() - 1 의 1 도 매직넘버 인가요??